app/assets/javascripts/forever_style_guide/account-popover.js

```diff
@@ -0,0 +1,42 @@
+// Trigger account popovers
+$(function () {
+  $('[data-toggle="popover-desktop"]').popover({
+    html: true,
+    content: function () {
+      return $('.account-popover').html();
+    },
+    template: '<div class="popover account-popover hidden-sm hidden-xs"><div class="arrow"></div><div class="popover-content"></div></div>',
```

Comment:
I think for now we'll live with it, something to think about post Forever Live perhaps...